Makefile

```diff
@@ -667,6 +671,10 @@ superclean: clean supercleanfiles
 		echo $(SUPERCLEAN_FILES) | tr ' ' '\n'; \
 		$(RM) $(SUPERCLEAN_FILES); \
 	fi
+	@ \
+	if [ -d $(CUB_DIR) ]; then \
```

Comment:
Hm, is there a reason we switched from `@- $(RM)...` syntax to this `if` block? Both should have the same effect (right?) and we're already using the former above...